src/components/index.js

```diff
@@ -10,3 +10,4 @@ Vue.component('font-awesome-layers', FontAwesomeLayers)
 Vue.component('c-permissions-button', components.CPermissionsButton)
 Vue.component('c-input-confirm', components.CInputConfirm)
 Vue.component('c-input-processing', components.CInputProcessing)
+Vue.component('c-item-picker', components.CItemPicker)
```

Comment:
I don't see any benefits in importing item picker as global component. And you are already importing it in the wrapping component, right?